src/Analysis/Ast/Test/DependencyResolverTests.cs

```diff
@@ -494,59 +494,6 @@ public void TestInitialize()
             walker.Remaining.Should().Be(0);
         }
 
-        [TestMethod]
-        public async Task ChangeValue_RemoveKeys() {
```

Comment:
Is this a case that won't pop up anymore?

Reply:
Yes. This is no longer needed.

Reply:
I should rephrase, would the test pass if it weren't removed?

Reply:
No, it wont as it was designed to remove random keys and the only case where we actually used it was complete reset - which was replaced by complete wipe out sans builtins.

Reply:
Ah, I see now. I didn't notice this test was using `RemoveKeys` (which you dropped in favor of a `Reset`).